addons/project/views/project_views.xml

```diff
@@ -810,6 +857,27 @@
             </field>
         </record>
 
+        <!-- Please update both act_window when modifying one (open_view_project_all_config or open_view_project_all_config_group_stage) as one or the other is used in the menu menu_project_config -->
+        <record id="open_view_project_all_config_group_stage" model="ir.actions.act_window">
+            <field name="name">Projects</field>
+            <field name="res_model">project.project</field>
+            <field name="domain">[]</field>
+            <field name="view_mode">tree,kanban,form,calendar,activity</field>
+            <field name="view_ids" eval="[(5, 0, 0),
+                (0, 0, {'view_mode': 'tree', 'view_id': ref('view_project')}),
+                (0, 0, {'view_mode': 'kanban', 'view_id': ref('view_project_kanban')})]"/>
```

Comment:
We rather like creating `ir.actions.act_window.view` records. The way you did is works, but make overrides difficult to handle as we need to rewrite all the records each time if you see what I mean.

Reply:
Done